tensorflow/lite/micro/kernels/fully_connected_test.cc

```diff
@@ -42,6 +42,17 @@ const float simple_weights_data[] = {
     1, 2, 3, 4, 5, 6, 7, 8, 9, 10,  // u = 1
     1, 2, 3, 4, 5, 6, 7, 8, 9, 10,  // u = 2
 };
+// INT4 isn't supported on Hexagon (b/258710417)
```

Comment:
let's make this `TODO(b/258710417): INT4 isn't currently supported on Hexagon.` That way the bug will have a link to all the TODO references in the code.

Reply:
Fixed.